components/RuiBreadcrumbs/src/RuiBreadcrumbs.ts

```diff
@@ -0,0 +1,67 @@
+/**
+* Copyright Deloitte Digital 2019
+*
+* This source code is licensed under the BSD-3-Clause license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import { LitElement, html, property, CSSResultArray, TemplateResult} from 'lit-element';
+import { variables, layout } from './RuiBreadcrumbs.css'
+
+
+export class RuiBreadcrumbs extends LitElement {
+	/**
+	 * The array of breadcrumbs
+	 */
+
+	@property({type : String})
```

Comment:
@grmillsy1 I'm not sure if it would help in this case, but I noticed you are treating the crumbs array as a string here and then attempting to parse it in the render method. If you have not already decided against it, possibly a better method would be to set the type as `@property({type : Array})`  or if that doesn't cover your use case exactly you can define a custom conversion via a converter function. 

The overhead of running JSON.parse inside the render function is not high, but in this case not necessary. 
It does mean the property would always be an array though, so instead of splitting the render logic based on `if (this.crumbs` you may need to convert to an empty array check instead. Let me know your thoughts :) 

More on the provided default property converters [here](https://lit-element.polymer-project.org/guide/properties#conversion-type).
More on custom conversions [here](https://lit-element.polymer-project.org/guide/properties#conversion-converter)

Reply:
@jnelssonsmith yeah I had some problems passing in the JSON array which Is why I passed it as a string, i'll re-visit this and let you know the outcome.

Reply:
@jnelssonsmith I am going to look into this in my next pull request so I can spend a bit more time on it! I hope thats ok. will let you know when I have done it

Reply:
@grmillsy1 As long as @jennasalau is okay with leaving it for now then I'm fine, make sure you leave a TODO comment or similar to remind yourself to investigate later 😄